fdbserver/include/fdbserver/IPager.h

```diff
@@ -221,6 +220,14 @@ class ArenaPage : public ReferenceCounted<ArenaPage>, public FastAllocated<Arena
 	uint8_t* rawData() { return buffer; }
 	int rawSize() const { return bufferSize; }
 
+	// Encryption key used to encrypt a page. Different encoding types may use different structs to represent
+	// an encryption key, and EncryptionKeyRef is a union of these structs.
+	struct EncryptionKeyRef {
```

Comment:
shouldn't this be a `union`, do we imagine both values to be set at the same time?

Reply:
I want to make it a union too, but a union don't have a default destructor, which cause further compilation error. I'll leave it for now without making more effort.